### PR TITLE
Availability and Meetings commands

### PR DESCRIPTION
**Pull request by fansxx**
basic command functionalities for availability and meeting. have not handled any exceptions yet

### CODE REVIEW

File: src/main/java/cooper/meetings/MeetingManager.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package cooper.meetings;
+
+import cooper.ui.Ui;
+
+import java.util.HashMap;
+
+public class MeetingManager {
+    private static HashMap<String, String> meetings = new HashMap(10);
```
Comment (theeugenechong) [inline src/main/java/cooper/meetings/MeetingManager.java:8]: Is the size fixed at 10 for now?

File: src/main/java/cooper/ui/Ui.java [hunk 2]
```diff
@@ -17,6 +17,7 @@
             +                  " \\_______/ \\______/  \\______/ |__/      \\_______/|__/      ";
 
     private static final String LINE = "=========================================================================";
+    private static final String TABLE_LINE = "-----------------------------------";
```
Comment (theeugenechong) [inline src/main/java/cooper/ui/Ui.java:20]: Noice 😙

**Review by theeugenechong — COMMENTED** [review]
LGTM! 👌🏻

File: src/main/java/cooper/meetings/MeetingManager.java (hunk 1 above)
Comment (fansxx) [reply src/main/java/cooper/meetings/MeetingManager.java:8]: yep for now!! im thinking of having a fixed number of keys corresponding to the timings later

**Comment by fansxx** [comment]
Fixes #17  
Fixes #16

**Pull request merged**